javaServices/coreJavaServices/src/joliex/util/TimeService.java

```diff
@@ -415,4 +422,38 @@ public Long getTimestampFromString( Value request )
 			throw new FaultException( "InvalidTimestamp", pe );
 		}
 	}
+
+	@RequestResponse
+	public String scheduleTimeout( Value request ) {
+		final String uuid = UUID.randomUUID().toString();
```

Comment:
UUID generation can be slow (requires random number generation). Wouldn't an AtomicLong suffice here instead of using UUIDs?